components/brave_wallet_ui/common/hooks/assets.ts

```diff
@@ -62,12 +62,7 @@ export default function useAssets (
       return []
     }
 
-    const nonZeroBalanceAssets = userVisibleTokensInfo.filter(token => {
-      const balance = getBalance(selectedAccount, token)
-      return balance !== '' && parseInt(balance) > 0
-    })
-
-    return nonZeroBalanceAssets.sort(function (a, b) {
+    return userVisibleTokensInfo.sort(function (a, b) {
```

Comment:
cc: @muliswilliam to check if this has any consequence for your other PR. Recall that we wanted to use `panelUserAssetList` for `sendAssetOptions`. I think filtering out all the zero-balance assets never made sense, so let's do the same for send assets list too.

Reply:
@onyb This is okay. As per our discussion, we need to agree on how to handle positive balances.